### PR TITLE
README V8.8 version number image update

### PR DESCRIPTION
**Pull request by mitchute**
Pull request overview
---------------------
Please change this line to a description of the pull request, with useful supporting information including whether it is a new feature, or fixes a defect, a cross reference to any defects addressed in this PR, the type of changes to be made, and whether diffs are expected/justified based on this change.

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [ ] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue.  The github issue should be referenced in the PR description
   - Refactoring: This pull request includes code changes that don't change the functionality of the program, just perform refactoring
   - NewFeature: This pull request includes code to add a new feature to EnergyPlus
   - Performance: This pull request includes code changes that are directed at improving the runtime performance of EnergyPlus
   - DoNoPublish: This pull request includes changes that shouldn't be included in the changelog

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [ ] Code style (parentheses padding, variable names)
 - [ ] Functional code review (it has to work!)
 - [ ] If defect, results of running current develop vs this branch should exhibit the fix
 - [ ] CI status: all green or justified
 - [ ] Performance: CI Linux results include performance check -- verify this
 - [ ] Unit Test(s)
 - C++ checks:
   - [ ] Argument types
   - [ ] If any virtual classes, ensure virtual destructor included, other things
 - IDD changes:
   - [ ] Verify naming conventions and styles, memos and notes and defaults
   - [ ] Open windows IDF Editor with modified IDD to check for errors
   - [ ] If transition, add rules to spreadsheet
   - [ ] If transition, add transition source
   - [ ] If transition, update idfs
 - [ ] If new idf included, locally check the err file and other outputs
 - [ ] Documentation changes in place
 - [ ] Changed docs build successfully
 - [ ] ExpandObjects changes?
 - [ ] If output changes, including tabular output structure, add to output rules file for interfaces

### CODE REVIEW

**Comment by nrel-bot-2** [comment]
@mitchute @lgentile it has been 14 days since this pull request was last updated.

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ This is the EnergyPlus Development Repository.  EnergyPlus™ is a whole buildin
  - Many users of EnergyPlus are active on [Unmet Hours](https://unmethours.com/), so that's a great place to start if you have a question about EnergyPlus or building simulation.
  - For more in-depth, developer-driven support, please utilize the [EnergyPlus Helpdesk](http://energyplus.helpserve.com/).
 
-# Releases [![](https://img.shields.io/badge/latest%20release-8.6.0-orange.svg)](https://github.com/NREL/EnergyPlus/releases/latest)
+# Releases [![](https://img.shields.io/badge/latest%20release-8.8.0-orange.svg)](https://github.com/NREL/EnergyPlus/releases/latest)
```
Comment (mjwitte) [inline README.md]: Or, do we want to remove the version number to prevent forgetting this in the future?

Comment (Myoldmopar) [reply README.md]: Yeah, unless we're able to do a dynamic badge there, it's probably better to just take the version out for now.  To keep the version number in there, we could look at getting the latest release number from energyplus.net and using that to build the badge, but that's probably a "later" thing.

**Comment by mitchute** [comment]
@mjwitte @Myoldmopar this should fix the issue permanently. It's not [OSU Orange](https://go.osuit.edu/communications/office/guidelines/visual/color_logo_usage) though, so maybe a non-starter for Mr. Lee

**Comment by Myoldmopar** [comment]
Ummm, throw a query parameter and get it back orange then :+1: 

`https://img.shields.io/github/release/NREL/energyplus.svg?colorB=FF7300` yields:



Great find on the automatic release shield though, @mitchute !!

**Comment by Myoldmopar** [comment]
Of course, if you invoke that hex code for the color on that badge (and I support it), need to explicitly change the other orange badges to that same hex code so they match.  Calling out "orange" as that query parameter did not override the autogenerated shield, so you need to use a hex code it seems.

**Review by Myoldmopar — APPROVED** [review]
:+1:

**Comment by mitchute** [comment]
That should do it.

**Comment by mjwitte** [comment]
Well, [Illini orange](http://creativeservices.illinois.edu/brand/logos-and-colors.html) would be preferable , but I'll merge this anyway.

**Pull request merged**